packages/plugin-types/package.json

```diff
@@ -16,5 +16,8 @@
   },
   "imports": {
     "#package.json": "./package.json"
+  },
+  "devDependencies": {
+    "execa": "^5.1.1"
```

Comment:
I think this should be a peerDependency since we are only referencing a type in our API, not actually using it in this package

Reply:
we have to install `execa` since it's a different version ☹️